phpunit.xml.dist

```diff
@@ -7,6 +7,7 @@
          processIsolation="false"
          stopOnFailure="false"
          bootstrap="tests/bootstrap.php"
+         cacheResult="false"
```

Comment:
Tested this variation a bit on phpunit7 - looks like it still works there. So adding this for phpunit8 seems good.